pkgs/desktops/gnome/default.nix

```diff
@@ -283,34 +291,34 @@ lib.makeScope pkgs.newScope (self: with self; {
 } // lib.optionalAttrs (config.allowAliases or true) {
 #### Legacy aliases
 
-  bijiben = gnome-notes; # added 2018-09-26
```

Comment:
We should just drop these aliases. Opened https://github.com/NixOS/nixpkgs/pull/154834.